torch/_torch_docs.py

```diff
@@ -2362,10 +2362,22 @@ def merge_dicts(*dicts):
 
 .. math::
     \text{out} = \text{abs} \cdot \cos(\text{angle}) + \text{abs} \cdot \sin(\text{angle}) \cdot j
+
+The behavior of this function is undefined if the absolute value is NaN, or if the angle is infinite.
+The absoulute value input is not validated, and the function will
+still perform the computation as described if it contains negative values,
+because that is the behavior defined by `std::polar`.
+
+.. note::
+    This implementation has the same exact logic as C++'s implementation of
```

Comment:
Suggestion: 

"torch.polar is similar to std::polar (link) and does not compute the polar decomposition of a complex tensor like Python's cmath.polar and SciPy's linalg.polar do. Like std::polar, this operation does not validate its inputs."

Reply:
I followed this suggestion.